x-pack/plugins/ml/public/application/model_management/expanded_row.tsx

```diff
@@ -130,6 +131,19 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
     description,
   } = item;
 
+  const inferenceStats = useMemo(() => {
+    if (!isPopulatedObject(stats.inference_stats)) return;
+
+    const result = { ...stats.inference_stats } as PartialBy<
+      Exclude<TrainedModelStat['inference_stats'], undefined>,
+      'cache_miss_count'
+    >;
+    if (item.model_type === 'pytorch') {
```

Comment:
```suggestion
    if (item.model_type ===TRAINED_MODEL_TYPE.PYTORCH) {
```

Reply:
updated in https://github.com/elastic/kibana/pull/160265/commits/91eb1cf10079e4bab2b9deb2dc256389ed04c7d5